base/libgit2/libgit2.jl

```diff
@@ -204,6 +204,23 @@ end
 
 Returns `true` if `a`, a [`GitHash`](@ref) in string form, is an ancestor of
 `b`, a [`GitHash`](@ref) in string form.
+
+# Example
+
+```julia
+julia> repo = LibGit2.GitRepo(repo_path);
+
+julia> LibGit2.add!(repo, test_file1);
+
+julia> commit_oid1 = LibGit2.commit(repo, "commit1")
```

Comment:
If this (and the other a couple lines down) display something at the REPL, you should either add output or a semicolon. Doesn't _really_ matter since it isn't a doctest, but it's more consistent.